assets/autocomplete/commandAutocompleteConfig.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2019-09/schema",
-  "$id": "http://www.openscope.co/assets/autocomplete/commandAutocompleteConfig.schema.json",
+  "$id": "http://www.openscope.io/assets/autocomplete/commandAutocompleteConfig.schema.json",
```

Comment:
@erikquinn Looks like our `.co` domain is still pointing to Heroku. When you have some time, could you update the dns to point to Render?